### PR TITLE
Add Windows Terminal Profiles plugin

### PR DESCRIPTION
**Pull request by paradox00**

### CODE REVIEW

**Comment by taooceros** [comment]
Mind adding filter functionality?

File: plugins.json [hunk 1]
```diff
@@ -552,5 +552,17 @@
...
+        "ID": "a56ae6ac-7b3b-47b9-857f-05ff25d29510",  
+        "Name": "Windows Terminal profiles",
+        "Description": "Windows Terminal profiles launcher",
+        "Author": "Aviv B.D.",
+        "Version": "0.0.2",
+        "Language": "python",
+        "Website": "https://github.com/paradox00/Flow.Launcher.Plugin.WindowsTerminal",
+        "UrlDownload":"https://github.com/paradox00/Flow.Launcher.Plugin.WindowsTerminal/releases/download/v0.0.2/Flow.Launcher.Plugin.WindowsTerminal.zip",
+        "UrlSourceCode": "https://github.com/paradox00/Flow.Launcher.Plugin.WindowsTerminal",
+        "IcoPath": "https://raw.githubusercontent.com/paradox00/Flow.Launcher.Plugin.WindowsTerminal/main/Images/app.png"
```
Comment (taooceros) [inline plugins.json:566]: Could you switch this to jsdelivr link?

**Comment by paradox00** [comment]
@taooceros  - What do you mean?

File: plugins.json (hunk 1 above)
Comment (paradox00) [reply plugins.json:566]: NP.

**Comment by taooceros** [comment]
> @taooceros - What do you mean filter functionality?

like I can search for the tilte with query. e.g. `wt powershell` should only show powershell as a result.

**Comment by paradox00** [comment]
@taooceros - I updated the plugin to work like that. Now `wt powershell` will return only results with powershell in the name.

**Comment by Garulf** [comment]
Hi, I noticed you have Flox commented out in your `requirements.txt`. If you have any issues or suggestions with Flox please feel free to post an issue on it's github page or hop on our Discord channel: https://discord.gg/tbSnTJQC

**Comment by Garulf** [comment]
Works great! 

There is a missing icon for the reload option i noticed while testing:

image

**Comment by paradox00** [comment]
The reload profiles was removed from the plugin, it does nothing as the profiles are loaded every time a RPC call is being made.

**Review by Garulf — APPROVED** [review]

**Pull request merged**